Source/ViewModels/DistanceLegendViewModel.js

```diff
@@ -195,10 +195,18 @@ function updateDistanceLegendCesium(_viewModel, scene) {
 
   if (defined(distance)) {
     let label;
-    if (distance >= 1000) {
-      label = `${(distance / 1000).toString()} km`;
+    if (units === 'nm') {
+      if (distance >= 1852) {
```

Comment:
These constants should be defined as a const above such as `const METERS_PER_KM = 1000` and `const  METERS_PER_NM = 1852`